sass/_QuerySuggestPreview.scss

```diff
@@ -1,5 +1,70 @@
 @import 'MagicBox/Colors';
 @import 'Variables';
+
+@mixin attachTo($attachment) {
+  position: absolute;
+  @if $attachment != bottom {
+    top: 0;
+  }
+  @if $attachment != top {
+    bottom: 0;
+  }
+  @if $attachment != right {
+    left: 0;
+  }
+  @if $attachment != left {
+    right: 0;
+  }
+}
+
+@mixin coveo-default-result-preview($maxImageHeightRelativeToWidth: 100%) {
```

Comment:
Why are we creating a mixin for `coveo-default-result-preview` (and `coveo-preview-results` at line 50) ?

It seems we are not reusing them anywhere except in one place, which means it's kind of adding complexity for no reason ?

Mixins are mostly useful in the same way a function in code is useful: to reduce repetition. Here it does not seems there's any repetition.

Reply:
Good point, just using `mixins` because I already made some parts of this CSS before and they had those `mixins`. As far as I remember, I used them because I saw a `mixin` being used for constants elsewhere and I wasn't sure if I should do the same.

I'll remove those `mixin`.